src/left-nav.jsx

```diff
@@ -225,6 +236,28 @@ const LeftNav = React.createClass({
       </div>
     );
   },
+  
+  _isOpen() {
```

Comment:
so... I should keep the `this.state.open` is sync with `this.props.open` in componentWillReceiveProps like in [dialog](https://github.com/callemall/material-ui/blob/master/src/dialog.jsx#L164), right?